Tests/GeoLib/TestPolygon.cpp

```diff
@@ -85,14 +85,14 @@ TEST_F(PolygonTest, isPntInPolygonCheckCorners)
 TEST_F(PolygonTest, isPntInPolygonCheckPointsRestOnPolygonEdges)
 {
     for (std::size_t k(0); k<_pnts.size()-1; k++) {
-        for (double t(0.0); t<1.0; t+=0.001) {
-            EXPECT_TRUE(_polygon->isPntInPolygon(
-                GeoLib::Point(
-                    (*_pnts[k])[0] + t*((*_pnts[k+1])[0]-(*_pnts[k])[0]),
-                    (*_pnts[k])[1] + t*((*_pnts[k+1])[1]-(*_pnts[k])[1]),
-                    (*_pnts[k])[2] + t*((*_pnts[k+1])[2]-(*_pnts[k])[2])
-                ))
-            );
+        double t = 0;
+        while (t < 1.0)
```

Comment:
This is basically the same as before. It's only called something else than `for`. Since 0.001 is not exactly representable as double
```
>>> print("{:.50f}".format(0.001))
0.00100000000000000002081668171172168513294309377670
```

it's not intuitively clear how many time the loop will iterate (1000 vs. 1001 times). This is not a big deal here,
but maybe using a true integer loop variable would be a better solution.